### PR TITLE
Perspective recommendation faq updates

### PR DESCRIPTION
**Pull request by suryanarayanan-venkatesan**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/faqs/cloud-cost-management-faqs.md [hunk 1]
```diff
@@ -259,6 +259,18 @@ Currently, there is no limit to the number of connectors per account.
 
 There is no restriction, but Harness recommends avoiding having the same connectors in FirstGen and NextGen.
 
+#### Do we support moving the recommendations from Applied to Open recommendations section ? 
+
+No. Currently we support only moving the recommendations from Open to applied section as of now
```
Comment (charanya-j) [inline docs/faqs/cloud-cost-management-faqs.md:264]:
```suggestion
No. Currently, CCM supports only moving the recommendations from the Open to the Applied tab.
```

File: docs/faqs/cloud-cost-management-faqs.md [hunk 2]
```diff
@@ -259,6 +259,18 @@ Currently, there is no limit to the number of connectors per account.
...
+
+#### Why are there no recommendations for GCP?
+
+We don’t support any GCP-specific recommendations as of today. We only have the support for the recommendations mentioned in the [doc](https://developer.harness.io/docs/category/recommendations)
```
Comment (charanya-j) [inline docs/faqs/cloud-cost-management-faqs.md:268]:
```suggestion
Currently, CCM doesn’t support any GCP-specific recommendations. For more information, go to [Recommendations](https://developer.harness.io/docs/category/recommendations).
```

File: docs/faqs/cloud-cost-management-faqs.md [hunk 3]
```diff
@@ -259,6 +259,18 @@ Currently, there is no limit to the number of connectors per account.
...
+
+#### How long do we show the recommendations ? 
+
+For Workload, Node Pool and Azure VM recommendations we will keep showing the recommendation for 7 days and for EC2 and ECS recommendations it will be 15 days. This is only for the recommendation details page. On the recommendation listing page we only show the recommendations which were updated in last 4 days.
```
Comment (charanya-j) [inline docs/faqs/cloud-cost-management-faqs.md:272]:
```suggestion
For Workloads, Node pools, and Azure VMs, CCM provides recommendations for 7 days, while for EC2 and ECS, it provides recommendations for 15 days. Note that these durations pertain to the recommendations details page. On the recommendations listing page, CCM displays only the recommendations updated in the last 4 days.
```

File: docs/faqs/cloud-cost-management-faqs.md [hunk 4]
```diff
@@ -280,9 +292,9 @@ The hourly granularity of cluster data is retained for 14 days. After 14 days, C
 
 ## On-premises
 
-#### Does CCM support on-premises platform installations?
+#### Does CCM support on-premises/Self Managed Platform (SMP) installations?
 
-Currently, no. Harness is working on providing this functionality.
+Yes. CCM supports the following features and functionalities in SMP environment. Connectors set up is supported only on AWS and Kubernetes as of now. Also we support Perspectives, Budgets, Scheduled Reports, Cost categories, Anomalies and Recommendations for now.
```
Comment (charanya-j) [inline docs/faqs/cloud-cost-management-faqs.md:297]:
```suggestion
Yes. CCM supports the following features and functionalities in the SMP environment for AWS and Kubernetes:
- Connector setup 
- Perspectives, 
- Budgets
- Scheduled Reports
- Cost categories
- Anomalies 
- Recommendations 
```

File: docs/faqs/cloud-cost-management-faqs.md [hunk 5]
```diff
@@ -356,6 +368,10 @@ To resolve any cost differences between the Perspective and Dashboard in Harness
 
   ![](./static/dashboard-time-filters.png)
 
+### How costs are displayed if the perspective contains costs which aren't associated with the grouped by identity ?
```
Comment (charanya-j) [inline docs/faqs/cloud-cost-management-faqs.md:371]:
```suggestion
### How are costs displayed if the perspective contains costs that aren't associated with the _grouped by_ identity?
```

File: docs/faqs/cloud-cost-management-faqs.md [hunk 6]
```diff
@@ -356,6 +368,10 @@ To resolve any cost differences between the Perspective and Dashboard in Harness
...
+We show `No` followed by `<group by>` for costs that don’t have any relation with the selected `<group by>`.
```
Comment (charanya-j) [inline docs/faqs/cloud-cost-management-faqs.md:372]:
```suggestion
CCM displays `No` followed by the <group by>` element for costs that don’t have any relation with the selected `<group by>` element.  For more information, go to [Create Perspectives](https://developer.harness.io/docs/cloud-cost-management/use-ccm-cost-reporting/ccm-perspectives/create-cost-perspectives#review-no-accountprojectetc).
```

File: docs/faqs/cloud-cost-management-faqs.md [hunk 7]
```diff
@@ -356,6 +368,10 @@ To resolve any cost differences between the Perspective and Dashboard in Harness
 
   ![](./static/dashboard-time-filters.png)
 
+### How costs are displayed if the perspective contains costs which aren't associated with the grouped by identity ?
+We show `No` followed by `<group by>` for costs that don’t have any relation with the selected `<group by>`. 
+
+Example : If a perspective contains rules for both AWS and GCP, and if the group by is done on GCP > SKU, the costs assosciated with other than GCP SKU will be displayed as No SKUs.
```
Comment (charanya-j) [inline docs/faqs/cloud-cost-management-faqs.md:374]:
```suggestion
For example, if a perspective contains rules for both AWS and GCP and if it is grouped by GCP > SKU, _No SKUs_ is displayed for costs (AWS, clusters, etc.) that don’t have any GCP SKUs associated with it.
```

File: docs/faqs/cloud-cost-management-faqs.md [hunk 8]
```diff
@@ -280,9 +292,9 @@ The hourly granularity of cluster data is retained for 14 days. After 14 days, C
 
 ## On-premises
 
-#### Does CCM support on-premises platform installations?
+#### Does CCM support on-premises/Self Managed Platform (SMP) installations?
```
Comment (charanya-j) [inline docs/faqs/cloud-cost-management-faqs.md:295]:
```suggestion
#### Does CCM support on-premises A.K.A Self-Managed Platform (SMP) installations?
```

File: docs/faqs/cloud-cost-management-faqs.md (hunk 4 above)
Comment (suryanarayanan-venkatesan) [reply docs/faqs/cloud-cost-management-faqs.md:297]: Added to the faq doc

File: docs/faqs/cloud-cost-management-faqs.md (hunk 3 above)
Comment (suryanarayanan-venkatesan) [reply docs/faqs/cloud-cost-management-faqs.md:272]: This is no longer valid so not adding it to the doc

File: docs/faqs/cloud-cost-management-faqs.md (hunk 1 above)
Comment (suryanarayanan-venkatesan) [reply docs/faqs/cloud-cost-management-faqs.md:264]: Added the suggestion to the faq

File: docs/faqs/cloud-cost-management-faqs.md (hunk 2 above)
Comment (suryanarayanan-venkatesan) [reply docs/faqs/cloud-cost-management-faqs.md:268]: Added a slight modification to the text as mentioned below.
Currently, CCM does not offer any GCP-specific recommendations. Our support is limited to the recommendations outlined in the [documentation](https://developer.harness.io/docs/category/recommendations).

Comment (charanya-j) [reply docs/faqs/cloud-cost-management-faqs.md:268]: This is grammatically correct. But we usually refrain from using "We" and "Our" in public documentation. It's better to say "Support is limited to the recommendations specified in the documentation."

**Pull request closed without merging**